decidim-core/app/controllers/decidim/newsletters_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Decidim
+  # The controller to handle the user's notifications dashboard.
+  class NewslettersController < Decidim::ApplicationController
+    skip_authorization_check
+
+    layout "decidim/mailer"
+    helper Decidim::SanitizeHelper
+    include Decidim::NewslettersHelper
+
+    def show
+      @organization = current_organization
```

Comment:
You should be able to use this helper method directly instead of the instance variable.

Reply:
@mrcasals We are using the same layout as the newsletter mailer "layouts/decidim/mailer.html.erb", and instance variables are being used. 
We have to options, create the instance variables on the controller or change/create a layout to see the newsletter on website, which one you prefer?

Reply:
Ah, I see. Then leave it as it is, sorry 😅